src/main/java/com/ksabbak/javaserver/StatusCode.java

```diff
@@ -0,0 +1,19 @@
+package com.ksabbak.javaserver;
+
+public enum StatusCode {
+    OK(200, "OK"),
+    NOT_FOUND(404, "Not Found");
+
+
+    public int code;
+    public String text;
+
+    private StatusCode(int code, String text){
+        this.code = code;
+        this.text = text;
+    }
+
+    public String getWholeStatus(){
```

Comment:
Is this function really getting the whole status or actually rendering the whole status as a string?